menu/menu_displaylist.c

```diff
@@ -5470,7 +5479,7 @@ unsigned menu_displaylist_build_list(
                   {
                      const char *ssid = scan->net_list[i].ssid;
                      if (menu_entries_append_enum(list,
-                              ssid,
+                              strlen(ssid) ? ssid : msg_hash_to_str(MSG_WIFI_EMPTY_SSID),
```

Comment:
I'd use `!string_is_empty(ssid)` here - `strlen()` will produce UB if `ssid` is NULL (even if that can't happen, it's better to be safe!)